templates/openshift-permissions.template.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: v1
```

Comment:
https://github.com/eriknelson/cdk-minishift-utils/blob/asb-update/addons/ansible-service-broker/ansible-service-broker.addon Looks like where this is going to get used. Could you add a comment to point to the project or some breadcumb as to who is using it.

Reply:
Good idea, will update.

Reply:
If it wasn't clear, I was hoping you would also add a comment in here to point to the `minishift-addons` repo. So that when I forget why this file exist I'll be pointed to that project and remember.